### PR TITLE
Fix node setup script issues

### PR DESCRIPTION
**Pull request by chihimng**
Ref #107

### CODE REVIEW

**Comment by chihimng** [comment]
cc @rickmak please help send to upstream if ok

**Review by hochiw — APPROVED** [review]

File: deploy/Makefile [hunk 1]
```diff
@@ -1,8 +1,6 @@
 STACK ?= validator
 CLOUD ?= azure
 
-include ./cmd/deploy/assets/Makefile
```
Comment (rickmak) [inline deploy/Makefile:4]: Why? `make -C deploy` is better than `make -C deploy/cmd/deploy/assets`

Comment (chihimng) [reply deploy/Makefile:4]: got relative path problems due to `include` another makefile, now manually proxy those commands at `deploy/Makefile`

File: SETUP.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ Following command will download a pre-compiled binary from Github. For compiling
 To setup a node using our setup script, please run the following command in the project root folder
 
 ```
-make -C deploy setup-node MONIKER=$MONIKER
+make -C deploy/cmd/deploy/assets setup-node MONIKER="<My Moniker>" GENESIS_URL="<URL>" LIKED_SEED_NODES="<SEED>"
```
[the remark this answers is not in the document]
Comment (chihimng) [reply SETUP.md]: refactor later? maybe make a script with predefined urls to different testnets

**Comment by rickmak** [comment]
Will close via https://github.com/likecoin/likecoin-chain/pull/61

**Pull request closed without merging**